_data/tools.yml

```
@@ -747,18 +747,18 @@
   v2: false
   v3: true
 
-- name: Vert.x Web API Service
+- name: express-openapi
   category: server
-  language: Java
-  link: https://vertx.io/docs/vertx-web-api-service/java/
-  github: https://github.com/vert-x3/vertx-web/
-  description: Create API service proxies using event bus with request/response validation
-  v2: false
+  language: Node.js / Typescript
+  link: https://www.npmjs.com/package/express-openapi
+  github: https://github.com/kogosoftwarellc/open-api/tree/master/packages/express-openapi
+  description: An unopinionated OpenAPI framework for Express, which supports Promise based middleware, response handlers and Security Filtering.
+  v2: true
   v3: true
 
 - name: BaucisJS + baucis-openapi3
   category: server
-  language: JavaScript
+  language: Node.js + Express + Baucis
```

```suggestion
  language: Node.js (Express)
```

Lets do this. Baucis is redundant as its in the name.